### PR TITLE
feat(agent): Add logPriority flag to agent chart

### PR DESCRIPTION
**Pull request by aroberts87**
## What this PR does / why we need it:

Add a new flag, logPriority, to the agent
chart to allow easier setting of the agent
console and file logging priorities in the
configmap

## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Title of the PR starts with type and scope, (e.g. feat(agent,node-analyzer,sysdig-deploy):)
- [x] Chart Version bumped for the respective charts
- [x] Variables are documented in the README.md (or README.tpl in some charts)
- [x] Check GithubAction checks (like lint) to avoid merge-check stoppers
- [x] All test files are added in the tests folder of their respective chart and have a "_test" suffix

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

**Review by ironashram — APPROVED** [review]
lgtm!

File: charts/agent/templates/_helpers.tpl [hunk 1]
```diff
@@ -318,3 +318,27 @@ Use global sysdig tags for agent
...
+        {{- end }}
+        {{- if (hasKey .Values.sysdig.settings.log "file_priority") }}
+            {{- $filePriority := .Values.sysdig.settings.log.file_priority }}
+            {{- $logConfigString = printf "%s\n  %s: %s" $logConfigString "file_priority" $filePriority }}
+        {{- end }}
+        {{- $logConfigString }}
+    {{- else if .Values.logPriority }}
+        {{- $logConfigString := "log:\n" }}
+        {{- $logConfigString = printf "%s  console_priority: %s\n  file_priority: %s\n" $logConfigString .Values.logPriority .Values.logPriority }}
+        {{- $logConfigString }}
```
Comment (lilx1ao) [inline charts/agent/templates/_helpers.tpl:342]: question: is this $logConfigString used for anything other than the console output?

Comment (aroberts87) [reply charts/agent/templates/_helpers.tpl:342]: It's being used to assemble the following block.
```
log:
  console_priority: <log_priority>
  file_priority: <log_priority>
```
I could combine the first two lines though at the expense of a longer overall line if you'd like. I wasn't sure if we had a character line limit or not so I split it up.

Comment (lilx1ao) [reply charts/agent/templates/_helpers.tpl:342]: I see, thanks. I overlooked line 55 from the configmap. this looks good 👍

**Review by lilx1ao — APPROVED** [review]

**Pull request merged**